### DISCUSSION TITLE
Error 500 on try showing temporary image upload

### DISCUSSION THREAD

**Discussion by fagnerdireito** [livewire/livewire#4063] [1. Help]
Hi everyone,

I have one form, where has a input file and upload to select image before upload. I can show this image on navigator but return one error 500 and not show the temporary image.

error:

        

how to solve this? 

![image](https://user-images.githubusercontent.com/2683236/138187811-24e7a421-392a-47fb-a2f9-43edfe398ff8.png)

**Comment by joshhanley** [comment]
@fagnerdireito hmm that seems unusual. Maybe check Livewire's temp directory to see if the file actually uploaded. Maybe it's a permissions issue on that folder?
